packages/consent/consent-tools/src/domain/validation/common-validators.ts

```diff
@@ -11,7 +11,7 @@ export function assertIsFunction(
 
 export function assertIsObject(
   val: unknown,
-  variableName: string
+  variableName = 'value'
```

Comment:
shouldn't it keep its typedef to `string`? won't TS restrict this to `value` only?

Reply:
Nope, it keeps the string type (not a const)